src/app/services/showcases/showcases.service.ts

```diff
@@ -1,33 +1,36 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import {  BehaviorSubject } from 'rxjs';
+import { BehaviorSubject, EMPTY, throwError, Observable } from 'rxjs';
 import { ShowcaseItem } from 'src/app/data/showcase-item';
-
+import { tap, catchError } from "rxjs/operators";
 
 @Injectable({
   providedIn: 'root'
 })
 export class ShowcasesService {
 
-  public showCasesSubject:BehaviorSubject<ShowcaseItem[]>=new BehaviorSubject<ShowcaseItem[]>(null);
-  SERVER_URL: string = "http://localhost:8080/api/";
-  constructor(private httpClient:HttpClient) {
-    this.getAllShowCases()
+  SERVER_URL: string = "api/showcases";
+  private showcasesSource = new BehaviorSubject([]);
```

Comment:
Change the BehaviorSubject's initial value to `null` instead of `[]` per @Yejielw 's suggestion